CentrED/Tools/DrawTool.cs

```diff
@@ -104,7 +104,10 @@ protected override void GhostApply(TileObject? o)
             {
                 if (o is StaticObject)
                 {
-                    return;
+                    if (!((MapManager.StaticFilterEnabled && MapManager.StaticFilterIds.Contains(o.Tile.Id) && !MapManager.StaticFilterInclusive) || o.Tile.Z < MapManager.MinZ || o.Tile.Z > MapManager.MaxZ))
```

Comment:
Can't we just use `MapManager.CanDrawStatic(StaticObject so)` here? Or does is do more than we need?

Reply:
yes, much better! done